tests/amqp_pause_test.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash -x
```

Comment:
I think this should be a python test. We have standardized on python as the language for tests. I don't think we should make an exception for this test.